src/components/navigation/navigation-item/navigation-item.component.ts

```diff
@@ -1,35 +1,21 @@
-import {
-    Component,
-    Input,
-    ContentChildren,
-    QueryList,
-    Renderer2,
-    ElementRef,
-    Optional,
-    SkipSelf,
-    Output,
-    AfterViewInit,
-    AfterContentInit,
-    OnDestroy
-} from '@angular/core';
-import { Router, NavigationEnd, UrlTree } from '@angular/router';
+import { ActivatedRoute, NavigationEnd, Router, RouterState, UrlTree } from '@angular/router';
+import { AfterContentInit, AfterViewInit, Component, ContentChildren, ElementRef, Input, OnDestroy, Optional, QueryList, Renderer2, SkipSelf, HostBinding } from '@angular/core';
 import { filter } from 'rxjs/operators/filter';
+import { Observable } from 'rxjs/Observable';
```

Comment:
I don't think this is used so can probably be removed

Reply:
Import removed.